d2bs/kolbot/libs/common/Precast.js

```diff
@@ -474,4 +475,23 @@ MainLoop:
 
 		return true;
 	};
+	
```

Comment:
This function is not needed, Precast already includes function: precastSkill which does the same thing as your energyshield function does.